### ISSUE TITLE
[Specification] Issues in new API Boilerplate chapter

### ISSUE THREAD

**Issue by krOoze** [KhronosGroup/Vulkan-Docs#283]
I found some issues in the new API Boilerplate chapter.

I made some line comments there(in the 1.0.20 commit log), but not sure if relevant people know to look there...

**Comment by oddhack** [comment]
I'll deal with some of these in the next update. Unclear what to do about the long #defines. The LaTeX code formatter does something reasonable but GNU source-highlight doesn't appear to have any such option, and using another highlighter would mean more nonstandard required components for the build environment. If you come up with a good portable option, feel free to reopen the issue and suggest it.

**Comment by krOoze** [comment]
> Unclear what to do about the long #defines. 

Spread them to multiple lines using line continuation `\` e.g. like so:
```
#if defined(__LP64__) || \
    defined(_WIN64) || \
    defined(__x86_64__) || \
    defined(_M_X64) || \
    defined(__ia64) || \
    defined (_M_IA64) || \
    defined(__aarch64__) || \
    defined(__powerpc64__)
#define VK_DEFINE_NON_DISPATCHABLE_HANDLE(object) typedef struct object##_T *object;
```

**Comment by krOoze** [comment]
Mostly fixed in 1.0.21.

Only overflowing code listing boxes remains. Since it's HTML, they will always overflow depending on browser window size. 

But the C code there is indeed too wide by any standards. It can be spread to multiple lines as shown above. The second overflow case is only a simple comment (can be fixed trivially).
